src/manipulators/strict_null_checks_manipulator.ts

```diff
@@ -531,7 +526,13 @@ export class StrictNullChecksManipulator extends Manipulator {
   ): void {
     const parent = statement.getParent();
 
-    if (parent && Node.isStatementedNode(parent)) {
+    if (
+      parent &&
+      Node.isStatementedNode(parent) &&
+      !statement.getLeadingCommentRanges().some(commentRange => {
```

Comment:
(Not for this change) if we deduct STRICT_NULL_CHECKS_COMMENT, may be need to throw an error or return a different status: so runner know he need to stop.